VOM_Fortran/VOM-code/modules.f90

```diff
@@ -185,7 +185,7 @@ module vegmod
       REAL*8  :: wsnew                  ! Total soil water store at next time step
       REAL*8  :: wsold                  ! Previous total soil water storage
 
-      REAL*8  :: o_pct = 0.300000d+00   ! Projected cover perennial vegetation (0-1)
+      REAL*8  :: o_cai = 0.300000d+00   ! Projected cover perennial vegetation (0-1)
```

Comment:
Why is a number provided here? Should this not be in an input file and/or optimised? Same in L194-197.

Reply:
I guess this is something coming from the past... it was always like this for all parameters, but actually we can remove the numbers is these are overwritten anyway by either the input-files or the sce-parameters. I just changed here the name of o_pct into o_cai, that's all

Reply:
Could you remove the numbers? Otherwise this is going to trip us and others up again.

Reply:
Will do!